front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js

```diff
@@ -219,14 +220,28 @@ export default function StandardAllowanceDefault(props) {
             </>
           )}
           <div className="col">
-            <InputGroupSNCF
-              id="standardAllowanceTypeSelect"
-              options={allowanceTypes}
-              handleType={handleType}
-              value={value.value}
-              type={value.type}
-              sm
-            />
+            {allowanceTypes.length > 1 ? (
+              <InputGroupSNCF
+                id="standardAllowanceTypeSelect"
+                options={allowanceTypes}
+                handleType={handleType}
+                value={value.value}
+                type={value.type}
+                sm
+              />
+            ) : (
+              <InputSNCF
+                id="standardAllowanceType"
+                label={allowanceTypes[0].label}
```

Comment:
Beware, allowanceTypes does not have a default value. This will throw an exception if allowanceTypes has a size of zero

Reply:
nor sure about the kind of display in case allowanceType is empty, but now it works

Reply:
done